lib/libc/armstdc/include/sys/timespec.h

```diff
@@ -9,11 +9,7 @@
 #define ZEPHYR_LIB_LIBC_ARMSTDC_INCLUDE_SYS_TIMESPEC_H_
 
 #include <sys/types.h>
-
```

Comment:
How does moving the timespec definition to another file help?

Reply:
The posix/time.h header includes <sys/_timespec.h>.  I think @cfriedt is probably better placed to explain what is going on here.  I was just mostly trying to get some testcases to build.

Reply:
oh, so you're just trying to make these headers compatible with other posix bits in Zephyr.  I don't know what the long term goal should be for that; it sounds like posix compatibility is considered an add-on in Zephyr, which is complicated when the C library provides existing posix APIs underneath Zephyr.